### PR TITLE
Fixed Markdown in headings of the Zoneminder binding documentation

### PR DESCRIPTION
**Pull request by DerEnderKeks**

### CODE REVIEW

**Comment by Confectrician** [comment]
Hello and thanks for your contribution! 👋 
Many parts of the documentation are [imported from other openHAB repositories](https://github.com/openhab/openhab-docs#automatically-generated-parts), e.g., the Binding- or add-on specific repository. This way we ensure that the articles are always in sync with the actual source code. This method has proven to be quite helpful to keep documentation up-to-date.
Long story short, you'll need to propose your change in the correct repository: 
https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.zoneminder/README.md

Thanks again for your willingness to contribute! 🙇
Please close this PR after you've created the new one.

**Comment by DerEnderKeks** [comment]
Alright! I created a new PR in the correct repository.

**Comment by Confectrician** [comment]
Thanks again for contributing.

For documentation:
openhab/openhab2-addons#2908
